media_driver/media_softlet/agnostic/gen12_base/codec/hal/dec/shared/hucitf/decode_huc_packet_creator_g12.h

```diff
@@ -48,6 +48,10 @@ class HucPacketCreatorG12 : public HucPacketCreatorBase
         MediaPipeline       *pipeline,
         MediaTask           *task,
         CodechalHwInterface *hwInterface);
+        
+    using HucPacketCreatorBase::CreateHucCopyPkt;
+    using HucPacketCreatorBase::CreateProbUpdatePkt;
+    using HucPacketCreatorBase::CreateStreamOutInterface;
```

Comment:
@StevenLi01 Could you help comment, why we define and use CodechalHwInterfaceNext in base class, but change it to CodechalHwInterface in child class and use legacy CodechalHwInterface everywhere in APO code?
Override class functions with different type of inputs seems not allowed by gcc13 now.